rpcserver.go

```diff
@@ -1024,10 +1064,11 @@ func (r *rpcServer) ListAssets(ctx context.Context,
 }
 
 func (r *rpcServer) fetchRpcAssets(ctx context.Context, withWitness,
-	includeSpent, includeLeased bool) ([]*taprpc.Asset, error) {
+	includeSpent, includeLeased bool,
+	queryFilters *tapdb.AssetQueryFilters) ([]*taprpc.Asset, error) {
 
 	assets, err := r.cfg.AssetStore.FetchAllAssets(
-		ctx, includeSpent, includeLeased, nil,
+		ctx, includeSpent, includeLeased, queryFilters,
```

Comment:
you should add some unit test coverage in `tapdb` to test the behavior of `AssetStore.FetchAllAssets`